src/msemu.c

```diff
@@ -600,6 +664,8 @@ int ms_init(ms_ctx* ms, ms_opts* options)
 	if (lcd_init(&ms->lcd_dat1bit, &ms->lcd_datRGBA8888, &ms->lcd_cas))
 		return MS_ERR;
 	if (io_init(&ms->io)) return MS_ERR;
+	ms_power_ac_set_status(ms, AC_GOOD);
```

Comment:
Change these to be evaluated from command line, default to AC_GOOD and BATT_HIGH though